snap7/client.py

```diff
@@ -49,55 +50,55 @@ def create(self):
         self._library.Cli_Create.restype = c_void_p
         self._pointer = S7Object(self._library.Cli_Create())
 
-    def destroy(self):
+    def destroy(self) -> Optional[int]:
         """
         destroy a client.
         """
         logger.info("destroying snap7 client")
         if self._pointer:
             return self._library.Cli_Destroy(byref(self._pointer))
         self._pointer = None
+        return None
 
-    def plc_stop(self):
+    def plc_stop(self) -> int:
         """
         stops a client
         """
         logger.info("stopping plc")
         return self._library.Cli_PlcStop(self._pointer)
 
-    def plc_cold_start(self):
+    def plc_cold_start(self) -> int:
         """
         cold starts a client
         """
         logger.info("cold starting plc")
         return self._library.Cli_PlcColdStart(self._pointer)
 
-    def plc_hot_start(self):
+    def plc_hot_start(self) -> int:
         """
         hot starts a client
         """
         logger.info("hot starting plc")
         return self._library.Cli_PlcHotStart(self._pointer)
 
-    def get_cpu_state(self):
+    def get_cpu_state(self) -> str:
         """
         Retrieves CPU state from client
         """
         state = c_int(0)
         self._library.Cli_GetPlcStatus(self._pointer, byref(state))
-
         try:
             status_string = cpu_statuses[state.value]
         except KeyError:
             status_string = None
```

Comment:
i think mypy infers  `status_string` can be both a string and a None following this logic, if you just raise the snap exception in this block directly it probably won't be confused anymore.

Reply:
Jep, that worked. :)